eq-author/src/components/modals/QuestionnaireSelectModal/index.js

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import styled from "styled-components";
+import { colors } from "constants/theme";
+import Modal from "components/modals/Modal";
+import Button from "components/buttons/Button";
+import ButtonGroup from "components/buttons/ButtonGroup";
+import ScrollPane from "components/ScrollPane";
+import Panel from "components/Panel";
+
+const ModalFooter = styled.div`
+  padding: 1.5em;
+  border-top: 1px solid ${colors.bordersLight};
+`;
+
+const StyledModal = styled(Modal)`
+  .Modal {
+    padding: 0;
+    width: 55em;
+  }
+`;
+
+const Container = styled.div`
+  background: white;
+`;
+
+const ModalTitle = styled.div`
```

Comment:
```suggestion
const ModalTitle = styled.h2`
```

This should be a heading tag to more accurately represent what it contains. Might need to update the styles when this is changed.

Chose h2 as h1 is the survey title in Author.